app/admin/page.tsx

```diff
@@ -84,7 +90,7 @@ export default async function AdminPage() {
   let prNumberError: string | null = null;
   
   try {
-    currentPRNumber = await getCurrentPRNumber();
+    currentPRNumber = await getCurrentPRNumber(environmentPrefix);
```

Comment:
Instead of passing in an environment prefix, just avoid calling this method at all if the environment is production